src/components/Article.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { useNavigate } from "react-router-dom";
+
+interface ArticleProps {
+  id: string;
+  title: string;
+  summary: string;
+}
+
+const Article: React.FC<ArticleProps> = ({ id, title, summary }) => {
```

Comment:
it's ok, but let's agree to write like this:
const Article = ({ id, title, summary }:ArticleProps) => {